script.js

```diff
@@ -4,33 +4,58 @@ const alertOverlay = document.querySelector("[data-id='alert-overlay']");
 const winTime = document.querySelector("[data-id='win-time']");
 const winMovements = document.querySelector("[data-id='win-movements']");
 const newGameButtons = document.querySelectorAll("[data-id='new-game-btn']");
+const backToMenuButtons = document.querySelector("[data-id='back-to-menu']");
 // timer
 const seconds = document.getElementById("seconds");
 // movements
 const movementsContainer = document.getElementById("movements");
 // welcome section
 const welcomePageSection = document.querySelector('[data-id = "welcome-page"]');
-const startGameButton = document.querySelector('[data-id = "start-game-btn"]');
+const easyLevelButton = document.querySelector('[data-id = "easy-level"]');
+const hardLevelButton = document.querySelector('[data-id = "hard-level"]');
 // game section
 const gameSection = document.querySelector("[data-id = 'game']")
-// navigate to game page
-startGameButton.addEventListener('click' , () => {
+
+// navigate to game page------------------------------------------------------
+
+easyLevelButton.addEventListener('click' , () => {
+  isEasyGame = true;
   welcomePageSection.style.display = "none";
   gameSection.style.display = 'block';
+  cardContainer.style.gridTemplateColumns ='repeat(4 , 1fr)'
```

Comment:
Styling like this is completely good in your case, but perhaps better to avoid in any other case. 
The main reason you would normally hear it is that it can quickly become difficult to debug and manage - when you have a designated css file, you expect all styles to be there. 
As said, not a problem here, just keep in mind for future 🙏🏽

Reply:
Ok thanks